### PR TITLE
coinmap for .coin

### PR DESCRIPTION
**Pull request by pembo210**

### CODE REVIEW

File: modules/cryptocoin.py [hunk 1]
```diff
@@ -121,9 +138,11 @@ def prettify(thing):
 @commandHook(['coin'])
 def coin(irc, ev):
     try:
-        coin = ev.args[0]
+        coin = coinmap[ev.args[0].lower()]
```
Comment (Polsaker) [inline modules/cryptocoin.py]: Isn't it easier if you use `coinmap.get(ev.args[0].lower(), even.args[0])`? You won't need the extra except block that way.

**Review by Polsaker — CHANGES_REQUESTED** [review]

File: modules/cryptocoin.py (hunk 1 above)
Comment (pembo210) [reply modules/cryptocoin.py]: The extra except block is for if the symbol doesn't match the coinmap symbols.

**Comment by Polsaker** [comment]
I know what it does. I said that it can be avoided if you use .get with the
two parameters

**Comment by pembo210** [comment]
neat

**Pull request merged**